identity/client/src/pages/roles/modals/EditModal.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+import { FC, useState } from "react";
+import TextField from "src/components/form/TextField";
+import { useApiCall } from "src/utility/api";
+import useTranslate from "src/utility/localization";
+import { FormModal, UseEntityModalProps } from "src/components/modal";
+import { Role, updateRole } from "src/utility/api/roles";
+import EntityList from "src/components/entityList";
+import usePermissions from "src/pages/roles/modals/usePermissions";
+
+const EditModal: FC<UseEntityModalProps<Role>> = ({
+  entity: role,
```

Comment:
FYI: I did not use desctucturing here because the variables `name`, `description` and `permissions` already exist, so you would need to name it like `currentName` etc... which in the end is longer than using role.name.